.github/workflows/deploy_docs_to_gh-pages.yml

```
@@ -0,0 +1,27 @@
+name: github pages
+
+on:
+  push:
+    branches:
+      - develop
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup mdBook
```

```suggestion
      - name: Install rust-docs with nightly toolchain
        uses: actions-rs/toolchain@v1
        with:
            profile: minimal
            toolchain: nightly
            override: true
            components: rust-docs
          
      - name: Setup mdBook
```

I think this needs to be run because the default toolchain is `stable`